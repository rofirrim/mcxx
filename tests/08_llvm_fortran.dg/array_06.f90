! <testinfo>
! test_generator="config/mercurium-fortran-llvm run"
! test_stdout="array_06.f90.out"
! </testinfo>
PROGRAM MAIN
    IMPLICIT NONE
    INTEGER :: A(3)
    INTEGER :: B(3)
    INTEGER :: C(3)
    INTEGER :: T(3)
    INTEGER :: I

    DO I = 1, 3
       A(I) = 2*I
       B(I) = I
       T(I) = 3*I
    END DO

    C = A + B

    PRINT *, A
    PRINT *, B
    PRINT *, C
    PRINT *, T
END PROGRAM MAIN
