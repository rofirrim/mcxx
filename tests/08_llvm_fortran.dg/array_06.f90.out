           2           4           6
           1           2           3
           3           6           9
           3           6           9
